src/clr-angular/layout/main-container/_layout.clarity.scss

```diff
@@ -7,6 +7,7 @@
     display: flex;
     flex-direction: column;
     height: 100vh;
+    min-height: 100vh;
```

Comment:
Was this broken due to applitools changes? Will this go away after rebase on master?

Reply:
ah yeah 😂 I forgot to remove this one. Thanks for catching this.